README.md

```diff
@@ -518,7 +518,28 @@ Create a base RHEL data volume prior to deploying OpenStack.  This will be used
     oc create -f compute.yaml
     ```
 
-6) (optional) Create roles file
+6) Node registration (register the overcloud systems to required channels)
+
+    Wait for the above resource to finish deploying (Compute and ControlPlane). Once the resources finish deploying proceed with node registration.
+
+    Use the procedure as described in [5.9. Running Ansible-based registration manually](https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html-single/advanced_overcloud_customization/index#running-ansible-based-registration-manually-portal) do do so.
+
+    NOTE: We recommend using manual registration as it works regardless of base image choice. If you are using
+    overcloud-full as your base deployment image then automatic RHSM registration could be used via the 
+    t-h-t rhsm.yaml environment role/file as an alternative to this approach. 
+
+    ```bash
+    oc rsh openstackclient
+    bash
+    cd /home/cloud-admin
+
+    <create the ansible playbook for the overcloud nodes - e.g. rhsm.yaml>
+
+    # register the overcloud nodes to required repositories
+    ansible-playpook -i /home/cloud-admin/ctlplane-ansible-inventory ./rhsm.yaml
```

Comment:
s/playpook/playbook

Reply:
But pook is so much better! Just kidding, will fix it